sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -5854,6 +5854,67 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val PIPELINES_STREAM_STATE_POLLING_INTERVAL: ConfigEntry[Long] = {
+    buildConf("spark.sql.pipelines.execution.streamstate.pollingInterval")
+      .doc(
+        "Interval at which the stream state is polled for changes. This is used to check " +
```

Comment:
Let's explicitly mention the time unit for the doc of the new time configurations.